src/index.js

```diff
@@ -8,14 +8,29 @@ module.exports = function () {
         testCount:      0,
         skipped:        0,
 
+        escapeXml (value) {
+            return this.replaceIllegalCharacters(this.escapeHtml(value));
+        },
+
+        escapeCdata (value) {
+            return this.replaceIllegalCharacters(value);
+        },
+
+        replaceIllegalCharacters (value) {
+            // eslint-disable-next-line no-control-regex
+            const RESTRICTED_CHAR = /[\x01-\x08\x0B-\x0C\x0E-\x1F\x7F-\x84\x86-\x9F]/g;
```

Comment:
Please move this line to the top of the file

Reply:
Done 👍